### PR TITLE
Bugfix/toggle in firefox - 1044

### PR DESCRIPTION
**Pull request by ckoutsiaris**
Closes sap/fundamental https://github.com/SAP/fundamental/issues/1044

Toggle should be ok now in firefox
screenshot 2018-12-06 at 13 11 36

#### Test

* http://localhost:4000/components/toggle.html

### CODE REVIEW

**Comment by joseegm** [comment]
Test Version for this PR was deployed

Built with commit 54df6ccb74026f33e6d9ab7f549f172ef487fda3

https://deploy-preview-1081--fundamental.netlify.com

File: test/templates/toggle/component.njk [hunk 1]
```diff
@@ -8,8 +8,10 @@ toggle:
 -->
 {% macro toggle(properties={}, modifier={}, state={}, aria={}, classes=[]) -%}
 {%- set _id = properties.id or utils.id() -%}
-<span class="fd-toggle{{ modifier.block | modifier('toggle') }}{{ classes | classes }}"{{ aria | aria }}>
-    <input type="checkbox" name="" value="" id="{{ _id }}"{{ ' disabled' if state.disabled }}{{ ' checked' if state.checked }}>
-    <span class="fd-toggle__switch" role="presentation"></span>
-</span>
+<label>
```
Comment (xak) [inline test/templates/toggle/component.njk]: This `label` cannot be here. It will be output as a duplicate when used in a `form__group`.

Comment (ckoutsiaris) [reply test/templates/toggle/component.njk]: If I don't add the label I don't have a way to test it in localhost:3030. Open to ideas :)

**Comment by xak** [comment]
I'm convinced this can be achieved with the `:before` and `:after` of the `.fd-toggle` and `.fd-toggle__switch` elements. We can continue to investigate.

**Pull request merged**